deploy/cr.yaml

```diff
@@ -2,3 +2,7 @@ apiVersion: "ingress.openshift.io/v1alpha1"
 kind: "ClusterIngress"
 metadata:
   name: "default"
+spec:
+  ingressDomain: cluster.local
```

Comment:
If we set a default ingress domain, it should be `${ROUTER_SERVICE_NAME.${ROUTER_SERVICE_NAMESPACE}.cluster.local`, which means we'll need to patch it in using Go code.

We can do this in a follow-up.

Reply:
This file is just some autogenerated thing that's not very useful... it should  be deleted and replaced with test fixtures somewhere